app/angular/src/server/angular-cli_config.js

```diff
@@ -31,16 +67,16 @@ export function getAngularCliWebpackConfigOptions(dirToSearch) {
     project.sourceRoot
   );
 
+  const projectRoot = path.resolve(dirToSearch, project.root);
+  const tsConfigPath = path.resolve(dirToSearch, 'src/tsconfig.app.json');
```

Comment:
Why exactly the tsconfig in `src`? What about the ones in maybe `projects/lib-a/tsconfig.lib.json`? 
I just ask to make sure it's not necessary.

Reply:
Mh, it might be that angular already has taken care of this use case. I've read the initial issue and there was a workaround where the issue opener added paths to the tsconfig. This is exactly what angular does when adding new projects:

```
{
  "compilerOptions": {
    [...]
    "paths": {
      "test-library": [
        "dist/test-library"
      ],
      "test-library/*": [
        "dist/test-library/*"
      ]
    }
  }
}
```

Reply:
Yes, but "paths" are not working with Storybook. You need to provide them as "alias" to webpack. Or to use for example - https://github.com/dividab/tsconfig-paths-webpack-plugin

It dosn't solve the "baseUrl" issue though like in #4136